src/network-provider/network-provider.registry.ts

```diff
@@ -5,7 +5,7 @@ export const DEFAULT_REGISTRY: Record<Exclude<Network, Network.BITCOIN_MAINNET>,
   [Network.AVALANCHE_MAINNET]: 'https://avalanche.public-rpc.com',
   [Network.BINANCE_SMART_CHAIN_MAINNET]: 'https://bsc-dataseed.binance.org/',
   [Network.CELO_MAINNET]: 'https://forno.celo.org',
-  [Network.ETHEREUM_MAINNET]: 'https://eth-mainnet-public.unifra.io',
```

Comment:
Did you encounter any issues with the previous RPC Url ?

Reply:
yes, didn't work for me. kept getting noNetwork errors in ethers.